src/test/java/org/springframework/integration/aws/config/xml/SqsOutboundChannedlAdapterParserTest.java

```diff
@@ -0,0 +1,47 @@
+package org.springframework.integration.aws.config.xml;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.beans.factory.BeanDefinitionStoreException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.integration.aws.outbound.SqsMessageHandler;
+import org.springframework.integration.test.util.TestUtils;
+
+/**
+ * @author Rahul Pilani
+ */
+public class SqsOutboundChannedlAdapterParserTest {
```

Comment:
Typo: SqsOutboundChanne~~d~~lAdapterParserTest 
Plus the rule: class for tests must have the `Tests` suffix